cypress/integration/tenants/add_tenant.spec.js

```diff
@@ -22,13 +22,17 @@ describe('Add Tenant', function () {
   })
 
   it('creates a tenant with a lease', function () {
+    cy.visit('add/property')
+    cy.wait(500)
+    cy.fillInPropertyForm()
+    cy.get('button').contains('SAVE').click()
+    cy.wait(500)
```

Comment:
We shouldn't use the UI to create a property if its needed for this test.

Rather than doing this. 

We can update line 7 to create a property.

```js
    cy.appFactories([
      ['create', 'staff', {email: 'staff@dwellingly.org', firstName: 'Franky', lastName: 'Bob'} ],
      ['create', 'property, {name: 'Property Name', address: 'address', ... } ]
    ])
```

Reply:
You don't need to pass any attributes. Only ones you want to override. Otherwise this file will add all the attributes automatically when a property is created.

`spec/factories/properties.rb`